tests/unit/src/main/scala/tests/TestingClient.scala

```diff
@@ -148,14 +169,32 @@ final class TestingClient(workspace: AbsolutePath, buffers: Buffers)
       .map(_.getMessage)
       .mkString("\n")
   }
-  private def toPath(filename: String): AbsolutePath =
-    TestingServer.toPath(workspace, filename)
+
   def workspaceMessageRequests: String = {
     messageRequests.asScala.mkString("\n")
   }
+
   def pathDiagnostics(filename: String): String = {
     pathDiagnostics(toPath(filename))
   }
+
+  private def toPath(filename: String): AbsolutePath =
+    TestingServer.toPath(workspace, filename)
+
+  def workspaceDiagnosticsCount: String = {
```

Comment:
Why is this moved? Maybe that is not neccessary like some other change sin this file.